osu.Game.Tests/Visual/Online/TestSceneWikiHeader.cs

```diff
@@ -82,7 +81,14 @@ private bool checkBreadcrumb()
             return result;
         }
 
-        private void dummyShowIndexPage() => wikiPageData.SetDefault();
+        private void dummyShowIndexPage()
```

Comment:
I'm a bit confused how this is related to the PR, can you explain?

Reply:
My bad, I messed up the commit order. This should be fixed the failed test because of this changes.

Reply:
Right, because `null` now means "error"?

Reply:
Yes